src/functions-and-arrays.js

```diff
@@ -1,73 +1,434 @@
 // Iteration #1: Find the maximum
-
+function maxOfTwoNumbers(num1, num2) {
+  if (num1 > num2) {
+    return num1;
+  } else if (num2 > num1) {
+    return num2;
+  } else {
+    return num1 && num2;
```

Comment:
Even if this this return (num1 && num2) doesn't alter the result. It also doesn't make much sense. You could either just pick one variable to return (the iterarion asks you to return any) or apply >= to one of the conditions above instead.